Specs/Number/Italian/OrdinalModel.json

```diff
@@ -1,59 +1,85 @@
 [
   {
     "Input": "tre trilionesimo",
-    "NotSupported": "dotnet",
-    "NotSupportedByDesign": "python,javascript,python",
+    "NotSupportedByDesign": "python,javascript,java",
     "Results": [
       {
         "Text": "tre trilionesimo",
         "TypeName": "ordinal",
         "Resolution": {
-          "value": "3000000000000",
-          "offset":"3000000000000",
+          "value": "3E+18",
+          "offset":"3E+18",
           "relativeTo":"start"
         }
       }
     ]
   },
   {
-    "Input": "trilionesimo",
-    "NotSupported": "dotnet",
+    "Input": "trentaseiesimo",
     "NotSupportedByDesign": "python,javascript,python",
     "Results": [
       {
-        "Text": "trilionesimo",
+        "Text": "trentaseiesimo",
         "TypeName": "ordinal",
         "Resolution": {
-          "value": "1000000000000",
-          "offset":"1000000000000",
+          "value": "36",
+          "offset":"36",
           "relativeTo":"start"
         }
       }
     ]
   },
   {
-    "Input": "cento trilioni",
```

Comment:
What happened to cases like this? Was it removed as invalid? Or just moved somewhere else in the spec that made it harder to find during review?
Please avoid moving test cases around (ideally, even keep them in an order as close to the source English as possible) if possible to make reviews and comparisons between cultures easier.

Reply:
this was modified to cento trilionesimo as it is in the English counterpart hundred trillionth.
The OrdinalModel reviewed commit includes reviews to match the English file order